test/services/CreditHistoryServiceSpec.scala

```diff
@@ -77,30 +77,32 @@ class CreditHistoryServiceSpec extends TestSupport with MockFinancialDetailsConn
 
     "an error is returned from the connector" should {
 
-      "return a credit history error (~getFinancialDetails failed)" in {
-        setupMockGetFinancialDetails(TaxYear.forYearEnd(taxYear), TaxYear.forYearEnd(taxYear + 1), nino)(FinancialDetailsErrorModel(500, "ERROR"))
-        TestCreditHistoryService.getCreditsHistoryV2(taxYear, nino).futureValue shouldBe Left(CreditHistoryError)
-      }
+      // TODO: we can drop this secrtion as these new methods not yet in use => getCreditsHistoryV2
```

Comment:
Are we keeping this in the code or is to be removed at a later date?

Reply:
code removed